custom_components/unifi_insights/coordinators/protect.py

```
@@ -424,7 +424,7 @@ def _mark_ws_frame_received(self, stream: str) -> None:
         self._recompute_ws_health_rollup()
 
     @callback
-    def _on_devices_connection_state_change(self, connected: bool) -> None:  # noqa: FBT001
+    def _on_devices_connection_state_change(self, connected: bool) -> None:
```

Ruff (0.16.3)

[warning] Boolean-typed positional argument in function definition

(FBT001)